lib/phase/flatten.js

```diff
@@ -13,7 +13,7 @@ const LABEL_MATCH = 7;
  * @param  {Object}   newJob    Job to be merged into. For ex: shared settings
  * @param  {Object}   oldJob    Job to merge. For ex: individual job settings
  */
-function merge(newJob, oldJob) {
+function merge(newJob, oldJob, fromTemplate) {
```

Comment:
Not sure what you mean? Like change the param to a JSON object with all these 3 things? Or you mean to change the name of this param?